src/main/java/it/smartcommunitylab/aac/webauthn/model/WebAuthnCredentialCreationInfo.java

```diff
@@ -0,0 +1,10 @@
+package it.smartcommunitylab.aac.webauthn.model;
+
+import com.yubico.webauthn.data.PublicKeyCredentialCreationOptions;
+
+public class WebAuthnCredentialCreationInfo {
+    public PublicKeyCredentialCreationOptions options;
```

Comment:
why does this class have all properties public?

Reply:
Changed to private, with getters and setters in 00e80afc51cf932110c94035a7846674ad0093fb